### ISSUE TITLE
Proxy server failed to send 

### ISSUE THREAD

**Issue by danbae56** [cloudpilot-emu/cloudpilot#19]
Please see [this Reddit post](https://www.reddit.com/r/Palm/comments/skhie1/cloudpilot_hotsync_problems/) for details.

**Comment by DirtyHairy** [comment]
I've posted the same reply on reddit:

I am not sure whether Palm Desktop for Mac ever supported network hotsync. I myself am running Palm Desktop in a (Parallels) Windows VM, there Hotsync manager offers me the options USB, serial, and network(TCP/IP).

<img width="645" alt="image" src="https://user-images.githubusercontent.com/2444224/152614472-a62ec16a-005b-4801-b137-7d2d802daf53.png">

"Manage hotsync name" can be found in the session settings --- you open those either by swiping the session to the right and selecting the icon on the left below it, or with a right-click on the session and selecting "Edit" from the menu (if you have a pointing device). This has no influence on the ability to hotsync, but this option will cause Cloudpilot to override the hotsync use name, which is undesired if you are hotsyncing.

<img width="453" alt="image" src="https://user-images.githubusercontent.com/2444224/152614518-3058c58c-267d-4de6-9f1f-99daaab3f049.png">

The "Permission denied" error is a bit weird; however, an error *is* expected as the Pilot can't connect to Palm desktop. However, I usually get Errno 32 - broken pipe.
 What address did you enter "Primary PC setup"? If you are curious, you can run the server with '--log debug', this will give you verbose information on where the Pilot tries to connect to. Still, I wouldn't worry too much about that message, I don't think that it is the reason for your error.

EDIT: I retried, and I now also got a "Permission denied" if the remote end is down. So, not so exotic after all.

If you are unsure of your setup, you can try Eudora Web and see whether you can access the internet. http://frogfind.com/ allows you to browse the web in a Palm-friendly manner.

To sum up, I am pretty sure that your problem is related to Palm Desktop --- it is quite possible that the MacOS version did not support TCP/IP. Maybe retry with a windows VM. VMWare fusion is free for personal use, and you can work with an unregistered windows trial for testing.

**Comment by DirtyHairy** [comment]
Following up on the discussion on reddit, I'll take the liberty to close this 😏
